omise/controllers/front/payment.php

```diff
@@ -0,0 +1,28 @@
+<?php
+if (! defined('_PS_VERSION_')) {
+    exit();
+}
+
+if (defined('_PS_MODULE_DIR_')) {
```

Comment:
@guzzilar The reason of this condition has been provided in the description of the commit, https://github.com/omise/omise-prestashop/commit/d0f5faea4f5a2438a7f307859c71d494bd376716.